gui/default/syncthing/core/connectivityStatusModalView.html

```diff
@@ -0,0 +1,54 @@
+<modal id="connectivity-status" status="{{connectivityStatusParams.status}}" icon="fas fa-fw {{connectivityStatusParams.type == 'listeners' ? 'fa-sitemap' : 'fa-map-signs'}}" heading="{{connectivityStatusParams.heading}}" large="no" closeable="yes">
+  <div class="modal-body" ng-switch="connectivityStatusParams.type">
+    <div ng-switch-when="listeners">
+      <div ng-if="listenersRunning.length > 0">
+	<p translate>
+	  The following network ports and protocols are available for other devices to connect to this one:
```

Comment:
Maybe use "listening" here on purpose to make it clear what listening means in this context for anyone confused about the term. Something like "Syncthing is listening on the following addresses for connection attempts from other devices:".

The "ports and protocols" formulation seems a bit out of place/too detailed. Below it's "ports or protocols", which isn't better. Both are just part of an address, and address is a less technical term that I'd consider more likely for any user to grasp. Basically I'd prefer using "address".